### ISSUE TITLE
Inaccurate project manager sorting

### ISSUE THREAD

**Issue by PlainCrown** [godotengine/godot#36127]
**Godot version:**
3.2

**OS/device including version:**
Windows 10

**Issue description:**
The project manager has three sorting options, one of which is named "Last Modified". However, this option actually sorts by the last project opened—even if there were no modifications to the project.

The first time I sorted by "Last Modified" after opening an older project just to look at code I had previously written, I was worried that I had accidentally modified something in that project and spent a while checking where I had made the mistake, even though my code and settings were the exact same.

Please rename the sorting type to "Last Opened" or truly make it sort by the last project that was modified in some way.

**Comment by KoBeWi** [comment]
The sorting is correct, the problem is that `project.godot` is modified just as you open the project. There aren't changes in the file, but the modification date is updated. IMO this is correct behavior.

**Comment by PlainCrown** [comment]
In that case, I don't think that simply opening and closing a project should warrant it appearing at the top of the "Last Modified" sorting list even if the modification date was updated.

It's my preferred way of sorting because the last project I worked on is always easy to find, but I also frequently revisit old projects to check on how I implemented a feature that I want to repeat in a new project. This results in the "Last Modified" sorting list being cluttered by projects that shouldn't be there.
